react-frontend/src/components/Learning/eventCard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import {
+  EventCardThumnail,
+  EventCardThumnailNullImg,
+  EventCardTextBox,
+  CardTitle,
+  CardDescription,
+  EventCardStyled,
+} from '../StyleComponents/card';
+import { HrefLinkStandalone } from '../StyleComponents/htmlTags';
+
+function EventCard({ name, description, eventbriteUrl, coverImgSrc }) {
+  return (
+    <EventCardStyled>
+      {/* <div style={{}} > */}
+      {coverImgSrc ? (
+        <EventCardThumnail src={coverImgSrc} data-testid="thumbnail" />
+      ) : (
+        <EventCardThumnailNullImg />
+      )}
+      <EventCardTextBox>
+        {/* <div style={{ padding: '24px', backgroundColor: 'white', display: 'flex', flexDirection: 'column', height: '100%' }}> */}
+        <CardTitle
+          style={{ fontSize: '25.92px', clear: 'both' }}
+          data-testid="title"
+        >
+          {name}
+        </CardTitle>
+        <CardDescription data-testid="description">
```

Comment:
There is noticeably more whitespace between the card description and link than in the wireframe:

Deployment:
![too-much-whitespace](https://user-images.githubusercontent.com/16946297/124511143-4521cb00-dd8a-11eb-975a-f03441c5ce31.png)

Wireframe:
![wireframe-less-whitespace](https://user-images.githubusercontent.com/16946297/124511154-4a7f1580-dd8a-11eb-826b-5de595a2e007.png)

Reply:
Removed the card description padding (reducing the spacing by 25px)